### PR TITLE
Symbol.species should on constructor, not instance

### LINKED ISSUE #50168: Symbol.species should in constructor, not instance
# Bug Report

### 🕗 Version & Regression Information

- This is a crash
- This changed between versions ______ and _______
- This is the behavior in every version I tried, and I reviewed the FAQ for entries about _________
- I was unable to test this on prior versions because _______

### 💻 Code

https://github.com/microsoft/TypeScript/blob/a3a5e009a5a7c95fbc0a08dff6d8947cceff7807/src/lib/es2017.sharedmemory.d.ts#L14

### PR DESCRIPTION
**Pull request by LongTengDao**
Fixes #50168

### CODE REVIEW

File: src/lib/es2017.sharedmemory.d.ts [hunk 1]
```diff
@@ -11,13 +11,13 @@ interface SharedArrayBuffer {
...
      */
     slice(begin: number, end?: number): SharedArrayBuffer;
-    readonly [Symbol.species]: SharedArrayBuffer;
     readonly [Symbol.toStringTag]: "SharedArrayBuffer";
 }
 
 interface SharedArrayBufferConstructor {
     readonly prototype: SharedArrayBuffer;
     new (byteLength: number): SharedArrayBuffer;
+    readonly [Symbol.species]: SharedArrayBuffer;
```
Comment (DanielRosenwasser) [inline src/lib/es2017.sharedmemory.d.ts:20]: Shouldn't this also be `SharedArrayBufferConstructor`?

Comment (LongTengDao) [reply src/lib/es2017.sharedmemory.d.ts]: @DanielRosenwasser Oh... Thank you!

**Review by DanielRosenwasser — APPROVED** [review]
Opening this up and Node.js and trying it at the console, I think this looks right.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
Seems like you need to accept the baselines so tests pass first.

**Comment by LongTengDao** [comment]
@DanielRosenwasser Thank you! I merged latest main from this repo, there is still error. Maybe I mistake your words "accept the baselines"? (I saw the error output has word "baseline" in the path, maybe "baseline" means other thing in this project?)

**Comment by jakebailey** [comment]
See: https://github.com/microsoft/TypeScript/blob/main/CONTRIBUTING.md#managing-the-baselines

**Comment by LongTengDao** [comment]
After study, I think I can't handle the error information above, sorry!

**Comment by jakebailey** [comment]
If you're not sure what to do, all that needs to be done is to run `gulp baseline-accept`, then commit the changed files.

**Comment by sandersn** [comment]
Do you want to keep working on this? Otherwise I'd like to close it to reduce the number of open PRs.

**Comment by LongTengDao** [comment]
> Do you want to keep working on this? Otherwise I'd like to close it to reduce the number of open PRs.

Sorry, I can't...

**Pull request closed without merging**